src/elexclarity/cli.py

```diff
@@ -55,17 +60,9 @@ def cli(electionid, statepostal, filename=None, style="default", outputType="res
         else:
             result = api_client.get_results(electionid, statepostal, **kwargs)
 
+    result = ClarityXMLConverter().convert(result, statepostal=statepostal)
 
-    '''
-    result = convert(
-        election_results,
-        electionID=electionID,
-        style=style,
-        outputType=outputType,
-        resultsBy=resultsBy,
-        **kwargs)
-    '''
-
-    if outputType in ["summary", "settings"]:
-        result = json.dumps(result, indent=2)
-    print(result)
+    # if outputType in ["summary", "settings"]:
```

Comment:
We should still check for `style=raw` and _not_ do a json dump if we're working with XML